setup.py

```diff
@@ -40,7 +40,7 @@
     packages=find_packages(),  # Finds modules with an __init__.py
     include_package_data=True,  # Pulls in non-module data from MANIFEST.in
     python_requires=">=3.5",
-    install_requires=["unidecode"],
+    install_requires=["unidecode", "bleach", "swapper"],
```

Comment:
asking users to manually install a required dependency is cumbersome. 
todo: pin packages versions.

Reply:
Good point, since bleach is a direct dependency that is one I should require directly in the project (but again I do try to minimize them).